### DISCUSSION TITLE
DefaultValue no EF6

### DISCUSSION THREAD

**Discussion by clt-pereira** [balta-io/forum#383] [Tira Dúvidas] [answered]
Balta, bom dia! tudo bem?

Estou utilizando o Entity Framework 6.0 com MVC 5 e estou tentando utilizar o data annotation DefaultValue(10) para que ao rodar a migração para o banco de dados, esse valor seja assumido como Default em uma coluna do tipo int, porém, ao executar a migração o valor default continua sendo o valor zero porque acredito queo EF 6.0 não dê suporte a essa annotation para refletir o valor no banco de dados.
Tentei a abordagem através do método OnModelCreating utilizando o método HasDefaultValue(10) mas pelo que vi aqui esse método não existe no Entity Framework 6.0.
Teria alguma dica para me dar de como fazer isso?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @clt-pereira por favor, poderia colocar os trechos de código do seu mapeamento, DbContext e alguns prints para evidenciar melhor o erro?
